account_bank_statement_import_move_line/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2017 Tecnativa - Luis M. Ontalba
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Bank statement import move lines',
+    'version': '12.0.1.0.0',
+    'category': 'Accounting',
+    'summary': 'Import journal items into bank statement',
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://www.tecnativa.com',
+    'depends': [
+        'account',
+    ],
+    'data': [
+        'wizards/account_statement_line_create_view.xml',
+        'views/account_bank_statement_view.xml',
+    ],
+    'license': 'AGPL-3',
+    'development_status': 'Stable',
```

Comment:
Proper `development_status` is `Production/Stable`. In fact, now there's not badge generated for status

Reply:
You need to declare `maintainers` for having this development status. Put me, you, or both.